docfiles/tracking.html

```diff
@@ -1,15 +1,17 @@
 <script type="text/javascript" src="/doccdn/pxtweb.js"></script>
 <script type="text/javascript">
     window.loadAppInsights = function (includeCookie) {
```

Comment:
remove includecookie?

Reply:
I still want to includeCookie for the case of pxtSkipAnalyticsCookie